controlpanel/frontend/jinja2/parameter-create.html

```diff
@@ -0,0 +1,108 @@
+{% from "input/macro.html" import govukInput %}
+{% from "fieldset/macro.html" import govukFieldset %}
+{% from "radios/macro.html" import govukRadios %}
+
+{% extends "base.html" %}
+
+{% set page_title = "Create a secret parameter for your webapp or airflow task" %}
+{% set legend -%}
+  Create a secret parameter for your webapp or airflow task
+{%- endset %}
+
+{% set page_name = "parameters" %}
+
+{% block content %}
+  <h1 class="govuk-heading-xl">{{ page_title }}</h1>
+
+  {% if form.error %}
+    <div class="error">{{ form.error }}</div>
+  {% endif %}
+
+  <form action="{{ url('create-parameter') }}" method="post">
+    {{ csrf_input }}
+    {% call govukFieldset({ "legend": { "html": legend|safe } }) %}
+      {{
+        govukInput({
+          "label": {
+            "text": "Key",
+            "classes": "govuk-label--m",
+          },
+          "classes": "govuk-!-width-two-thirds",
+          "hint": {
+            "text": '60 chars max, only letters, numbers and underscores'
+          },
+          "name": "key",
+          "attributes": {
+            "pattern": "[a-zA-Z0-9_]{1,60}",
+            "maxlength": "60",
+          },
+          "value": form.key.value(),
+          "errorMessage": { "html": form.key.errors|join("<br>") } if form.key.errors else {}
+        })
+      }}
+      {{
+        govukInput({
+          "label": {
+            "text": "Value",
+            "classes": "govuk-label--m",
+          },
+          "classes": "govuk-!-width-two-thirds",
+          "hint": {
+            "text": 'String. Any characters'
+          },
+          "name": "value",
+          "attributes": {
+          },
+          "value": form.value.value(),
+          "errorMessage": { "html": form.value.errors|join("<br>") } if form.value.errors else {}
+        })
+      }}
+      {{
+        govukInput({
+          "label": {
+            "text": "Role name",
+            "classes": "govuk-label--m",
+          },
+          "classes": "govuk-!-width-two-thirds",
+          "hint": {
+            "text": '60 chars max, only lowercase letters and underscores'
+          },
+          "name": "role_name",
+          "attributes": {
+            "pattern": "[a-z_]{1,60}",
+            "maxlength": "60",
+          },
+          "value": form.role_name.value(),
+          "errorMessage": { "html": form.role_name.errors|join("<br>") } if form.role_name.errors else {}
+        })
+      }}
+      {{
+        govukRadios({
+          "label": {
+            "text": "Airflow or Web app",
```

Comment:
This field is bad UX. Better to have the form on the app page and populate this field automatically.

Reply:
How can the app know where the param is going to be used tho? At least it's a radio button so it shouldn't allow them to enter weird stuff here.

Reply:
This field (if it is actually necessary), should be populated automatically depending on whether the user is creating a Parameter from a webapp page or a (future) Airflow role/dag page.

Reply:
For now we still need to know if it is airflow/webapp for restricting access to airflow `iam_policy.json` files. But yes, I agree that if/when we have an airflow page we should set it automatically.